README.md

```diff
@@ -59,7 +59,7 @@ The SDK currently supports and is tested on the following OS/Compiler variations
 - 64bit Linux with clang 9
 - 32bit Linux with GCC 7 (cross compiled from 64bit host)
 - 64bit Windows with MSVC 2019
-- 32bit Windows with MSVC 2017
+- 64bit Windows with MSVC 2022
```

Comment:
We are running the 32bit version on Windows 2019 though.
Also, is `windows-latest` coming with VS2022? I wasn’t aware, but thats nice.